build/js/live-editor.output.js

```diff
@@ -529,11 +529,13 @@ window.LiveEditorOutput = Backbone.View.extend({
             // Normal case
             } else {
                 // This is debounced (async)
-                this.test(userCode, this.validate, errors, function(errors, testResults) {
-                    this.results.errors = errors;
-                    this.results.tests = testResults;
-                    this.phoneHome();
-                }.bind(this));
+                if (this.validate !== "") {
+                    this.test(userCode, this.validate, errors, function(errors, testResults) {
+                        this.results.errors = errors;
+                        this.results.tests = testResults;
+                        this.phoneHome();
+                    }.bind(this));   
+                }
```

Comment:
This isn't necessary for resource caching.  It should've been included in the scrubber performance pull request.  I'm including it here because I don't want to forget about.